### PR TITLE
Refactor(cli): extract cli flags from proxy and provider pkg

### PR DESCRIPTION
**Pull request by zoumo**
- delete useless cli flags
- extract cli flags from proxy and provider pkg
- add new cli flags `additional tolerations`
- change release to v0.1.1

### CODE REVIEW

**Comment by kdada** [comment]
What's the context?

**Comment by zoumo** [comment]
The context is that I add new flag `additional tolerations` to let pods tolerate additional taints.
And optimize the cli flags by the way.

**Comment by zoumo** [comment]
@kdada PTAL

File: config/config.go [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+type additionalTolerations []string
+
+func (a *additionalTolerations) Set(value string) error {
+	values := strings.Split(value, ",")
+	if len(values) == 0 {
+		return nil
+	}
+	*a = append(*a, values...)
+	// set key here
+	toleration.AdditionalTolerationKeys = *a
```
Comment (kdada) [inline config/config.go]: Add a `Set` function to package `toleration`

**Pull request merged**